### PR TITLE
feat: add support for reserved instances #148

### PR DESCRIPTION
**Pull request by kevcube**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -382,3 +382,26 @@ variable "restore_to_point_in_time" {
   description = "An object specifying the restore point in time for the DB instance to restore from. Only used when `snapshot_identifier` is not provided."
   default     = null
 }
+
+variable "use_reserved_instances" {
```
Comment (Gowiem) [inline variables.tf:386]: Cloud posse convention (and ours too!): Use `_enabled` for `bool` type variables. [See here](https://docs.cloudposse.com/best-practices/terraform/#use-positive-variable-names-to-avoid-double-negatives).

**Review by Gowiem — CHANGES_REQUESTED** [review]
@kevcube one best practice to follow. Also, regarding your comment: 

> bats is failing because not all outputs have descriptions but I am not seeing any outputs without descriptions.

[See lines here in `examples/complete`](https://github.com/cloudposse/terraform-aws-rds/blob/main/examples/complete/outputs.tf#L41-L51). Fix those and request me as a reviewer and I'll give it another once through.

**Comment by Gowiem** [comment]
@kevcube see my review comment for how to fix bats tests

**Comment by kevcube** [comment]
@Gowiem I see no review comment 👀

**Comment by Gowiem** [comment]
image

@kevcube

**Comment by kevcube** [comment]
oh duh. ok thanks, fixed.

**Pull request closed without merging**